addons/web_editor/models/ir_attachment.py

```diff
@@ -42,7 +42,10 @@ def _compute_image_src(self):
                 continue
 
             if attachment.type == 'url':
-                if attachment.url.startswith('/'):
+                # Only check if the url starts with a '/' is not enough to
+                # define that it is a local url as it could also be an external
+                # protocol relative url.
+                if attachment.url.startswith('/') and not attachment.url.startswith('//'):
```

Comment:
Actually, maybe is it better to change this condition with something like
```py
 if re.match(r'^/\w+/static/', attachment.url)
```
to be consistent with what is done in the `/web_editor/modify_image/` route?